dmutils/forms/validators.py

```diff
@@ -10,3 +17,60 @@ class EmailValidator(Regexp):
     def __init__(self, **kwargs):
         kwargs.setdefault("message", "Please enter a valid email address.")
         super(EmailValidator, self).__init__(self._email_re, **kwargs)
+
+
+class GreaterThan:
+    """
+    Compares the values of two fields.
+
+    :param fieldname:
+        The name of the other field to compare to.
+
+    :param message:
+        Error message to raise in case of a validation error.
+    """
+    def __init__(self, fieldname, message=None):
+        self.fieldname = fieldname
+        self.message = message
+
+    def __call__(self, form, field):
+        try:
+            other = form[self.fieldname]
+        except KeyError:
+            raise ValidationError(field.gettext("Invalid field name '%s'." % self.fieldname))
+        if other.data and not field.data > other.data:
+            d = {
+                'other_label': hasattr(other, 'label') and other.label.text or self.fieldname,
+                'other_name': self.fieldname
+            }
+            message = self.message
+            if message is None:
+                message = field.gettext('Field must be greater than %(other_name)s.')
+
+            raise ValidationError(message % d)
+
+
+class Y2K:
+    """
+    Prepare for the 21st Century.
```

Comment:
*I* like this joke but it might not make much sense for people who are too young to know what relevance "Y2K" has to for digit years...

On this note, is it worth maybe including this validator in the `DateField` by default?

Reply:
Kids these days... If we decide to keep the class I will rename it to `FourDigitYear`.

I thought about including the four digit year check as part of the processing, but the reasons I kept it as a validator were that a) you can't then set the validation error message in a way that is consistent with WTForms, and b) by including the validator explicitly it acts as documentation saying that this date must have a four digit year.

On the other hand, I don't think we'd ever want to accept a two digit year. Thoughts @galund?